src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -14,9 +14,7 @@
 
 package codeu.model.store.persistence;
 
-import codeu.model.data.Conversation;
-import codeu.model.data.Message;
-import codeu.model.data.User;
+import codeu.model.data.*;
```

Comment:
Wow! Didn't know you can do that, that's cool!

Reply:
I did not know that either; I think the Google Formatter did that; it looks neat.